pkgs/by-name/ma/maturin/package.nix

```diff
@@ -24,10 +24,12 @@ rustPlatform.buildRustPackage rec {
 
   cargoHash = "sha256-yLKt/Xml7ig6QG3T5Qn39tW7U5NIN1hSOaLiSRMiy5I=";
 
-  buildInputs = lib.optionals stdenv.hostPlatform.isDarwin [
-    darwin.apple_sdk.frameworks.Security
-    libiconv
-  ];
+  buildInputs =
+    [ python3 ]
```

Comment:
It shouldn't depend on specific python version...

Reply:
Correct. This is not necessary.